iceprog/Makefile

```diff
@@ -1,6 +1,20 @@
 include ../config.mk
-LDLIBS = -L/usr/local/lib -lm
-CFLAGS = -MD -O0 -ggdb -Wall -std=c99 -I/usr/local/include
+
+LIBFTDI_VERSION = $(shell $(PKG_CONFIG) --modversion libftdi1 2>/dev/null)
+ifneq ($(LIBFTDI_VERSION),)
+  LIBFTDI_NAME = ftdi1
+else
+  LIBFTDI_NAME = ftdi
+endif
+
+UNAME := $(shell uname -s)
+ifneq ($(UNAME),Darwin)
```

Comment:
I'd actually prefer `ifneq ($(shell uname -s),Darwin)` since `$(UNAME)` isn't used anywhere else and `shell uname -s` is a pretty short string.